packages/react/src/components/DateTimePicker/DateTimePicker.jsx

```diff
@@ -793,9 +816,9 @@ const DateTimePicker = ({
                           onChange={onRelativeLastNumberChange}
                           translateWithId={(messageId) =>
                             messageId === 'increment.number'
-                              ? `${i18n.increment} ${i18n.number}`
+                              ? `${strings.increment} ${strings.number}`
```

Comment:
it would be nice to rename `strings` to `mergedI18n`. Just to make it more consistency with other components. Also `strings` is kind of data type, so we may get confused.

Reply:
> it would be nice to rename `strings` to `mergedI18n`. Just to make it more consistency with other components. Also `strings` is kind of data type, so we may get confused.

Thanks for the feedback. I've update all `strings` to `mergedI18n`.